src/rules/consistent-test-it.ts

```diff
@@ -39,6 +50,10 @@ export default {
       CallExpression(node) {
         const nodeName = getNodeName(node.callee);
 
+        if (!nodeName) {
+          return;
```

Comment:
new, but seeing as tests pass I guess it doesn't matter.

Reply:
It's required by typescript - `getNodeName` could always return null:

```
export const getNodeName = node => {
  function joinNames(a, b) {
    return a && b ? `${a}.${b}` : null;
  }

  switch (node && node.type) {
    case 'Identifier':
      return node.name;
    case 'MemberExpression':
      return joinNames(getNodeName(node.object), getNodeName(node.property));
  }

  return null;
};
```

Ah, the joys of complex ASTs :joy:

Reply:
> seeing as tests pass I guess it doesn't matter.

I believe this is b/c of the `isDescribe` - the conditions for which `getNodeName` would return `null` are mutually exclusive w/ `isDescribe`, which is why this has never been a problem.

Reply:
Just so it's written down somewhere, this is the code that can be used to make `getNodeName` to return `null`:

```
[1,2,3].forEach(() => { test("foo") })
```

It's b/c `[1,2,3]` is an `ArrayExpression`, which causes `getNodeName` to bail out.

I needed to know how to do this to implement a test for another rule to get coverage other the line :)